easybuild/tools/github.py

```diff
@@ -958,6 +958,32 @@ def not_eligible(msg):
     return res
 
 
+def list_prs(parameters):
+    """
+    List PRs
```

Comment:
I'd reword this to `List pull requests according to specified selection/order parameters`